pai-management/bootstrap/prometheus/node-exporter-ds.yaml.template

```diff
@@ -33,6 +33,12 @@ spec:
       containers:
       - image: prom/node-exporter:v0.15.2
         imagePullPolicy: Always
+        readinessProbe:
+          httpGet:
+            path: /metrics
+            port: {{ clusterinfo['prometheusinfo']['node_exporter_port'] }}
```

Comment:
Are there will be any server to handle this http request?

Reply:
promethueus will fetch this address to get the metrics from every node